### DISCUSSION TITLE
FailedToExecuteCommand `"gswin64c.exe" / Unable to add as a reference

### DISCUSSION THREAD

**Discussion by jpmalexandrino** [dlemstra/Magick.NET#1352] [Help]
Hello,

I'm trying to develop an extension to be used in outsystems, using ImageMagick to convert pdf files to an image. I'm getting this error:

"FailedToExecuteCommand `"FailedToExecuteCommand `"gswin64c.exe" -q -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 "-sDEVICE=pngalpha" -dTextAlphaBits=4 -dGraphicsAlphaBits=4 "-r300x300" -dPrinted=false  "-sOutputFile=C:/Users/OutSystemsApplications/AppData/Local/Temp/magick-vKrFP7M2f-PcFuB-yuLyhys1xfPnDXIi%d" "-fC:/Users/OutSystemsApplications/AppData/Local/Temp/magick-j-IwY4GZU2rlolQoIEJqwlKQ6JxFiUOU" "-fC:/Users/OutSystemsApplications/AppData/Local/Temp/magick-cmedUtASX4hVBC7AifkesGd9wn4LT2hb"' (The system cannot find the file specified.
) @ error/delegate.c/ExternalDelegateCommand/517"

I cannot install the Ghostscript in my PC or even in the server. I'm trying to put the files as a reference in the project but I'm also getting error:

![image](https://user-images.githubusercontent.com/126629287/222012551-57ddeba3-88d6-4b5e-b7bd-62d341474f4e.png)

Does someone have an idea how to solve this?

**Comment by dlemstra** [comment]
I hope you are aware that you need to pay for a commercial license when you want to use Ghostscript in a commercial product. And you can find more information about getting it working here: https://github.com/dlemstra/Magick.NET/tree/9a79cebc10e86240e46c795efb5a6863c2cd7402/docs#ghostscript.
